chapter4/lambda_function.py

```diff
@@ -0,0 +1,72 @@
+import json
+import base64
+import boto3
+import os
+import mlflow
+import numpy as np
+
+
+session = boto3.Session()
+credentials = session.get_credentials()
+
+# Access individual credentials
+AWS_ACCESS_KEY_ID = credentials.access_key
+AWS_SECRET_ACCESS_KEY = credentials.secret_key
+AWS_REGION = session.region_name
+
+
+os.environ['AWS_S3_ENDPOINT_URL'] = f'https://s3.{AWS_REGION}.amazonaws.com'
+
+# MLflow setup
+MLFLOW_TRACKING_URI = "http://127.0.0.1:5000"
+mlflow.set_tracking_uri(MLFLOW_TRACKING_URI)
+
+RUN_ID = "e308ab2a149249a4b161cb428b4abc23"
```

Comment:
your run id should also be stored in an environment variable

Reply:
Changes were made in chapter4 before merging and rebasing chapter5. Will go through now in chapter5 to identify and fix all such cases.